src/systems/ec/specs/components/language-list/demo/data--splash.js

```diff
@@ -0,0 +1,30 @@
+// Simple content for demo
```

Comment:
Languages order should follow https://github.com/ec-europa/europa-component-library/blob/master/src/generic/generic-component-language-list/languages.json